frameworks/FSharp/frank/src/App/Program.fs

```diff
@@ -0,0 +1,126 @@
+module App.App
+
+open System
+open System.IO
+open System.Threading.Tasks
+open Microsoft.AspNetCore.Hosting
+open Microsoft.AspNetCore.Http
+open Dapper
+open Frank.Builder
+open FSharp.Control.Tasks
+open Npgsql
+open App
+open Models
+
+let private DefaultCapacity = 1386
+let private MaxBuilderSize = DefaultCapacity * 3
+let private BufferSize = 27
+
+type MemoryStreamCache = 
+    
+    [<ThreadStatic>]
+    [<DefaultValue>]
+    static val mutable private instance: MemoryStream
+
+    static member Get() = MemoryStreamCache.Get(DefaultCapacity)
+    static member Get(capacity:int) = 
+        
+        if capacity <= MaxBuilderSize then
+            let ms = MemoryStreamCache.instance;
+            let capacity = max capacity DefaultCapacity
+            
+            if ms <> null && capacity <= ms.Capacity then
+                MemoryStreamCache.instance <- null;
+                ms.SetLength 0L
+                ms
+            else
+                new MemoryStream(capacity)
+        else
+            new MemoryStream(capacity)
+
+    static member Release(ms:MemoryStream) = 
+        if ms.Capacity <= MaxBuilderSize then
+            MemoryStreamCache.instance <- ms
+
+let inline contentLength x = new Nullable<int64> ( int64 x )
+
+let json' data : HttpContext -> Task<unit> =
+    let bytes = Utf8Json.JsonSerializer.Serialize(data)
```

Comment:
@panesofglass  if I put a `printfn "test"` here and then hit the json route over and over again, it will not print. If it is moved into the function, it will, so this is not currently meeting the requirements.

Reply:
You are correct. I thought it was under the `fun ctx -> ...` bit. I have updated it so that it serializes on each request.